scripts/reformat-cmake

```diff
@@ -29,18 +29,9 @@ if [ -z "$(which sponge)" ]; then
 	exit 1
 fi
 
-for file in $(
-	find . \( -name 'CMakeLists.txt' -or -name '*.cmake' \) -and \
-		\( \
-		-path './benchmarks/*' \
-		-or -path './cmake/*' \
-		-or -path './doc/*' \
-		-or -path './examples/*' \
-		-or -path './scripts/*' \
-		-or -path './src/*' \
-		-or -path './tests/*' \
-		-or -path './CMakeLists.txt' \
-		\)
-); do
-	$CMAKE_FORMAT "$file" | unexpand | sponge "$file"
+directories="benchmarks cmake doc examples scripts src tests"
```

Comment:
Why is directories needed? git-ls-files should already do the correct thing.

Reply:
The old script only checked these directories, but it probably isn't required anymore.